apisix/plugin.lua

```diff
@@ -761,8 +770,11 @@ local function check_single_plugin_schema(name, plugin_conf, schema_type, skip_d
     end
 
     if plugin_obj.check_schema then
-        local disable = plugin_conf.disable
-        plugin_conf.disable = nil
+        local disable = check_disable(plugin_conf)
+        if disable ~= nil then
+            plugin_conf._meta.disable = nil
```

Comment:
Once we have moved the `disable` to _meta, we don't need to delete it during the check_schema. As there is no longer a top level extra field in the conf.

Reply:
fixed